src/resources/cooperative.ts

```diff
@@ -61,6 +61,61 @@ export const CooperativeResource: ResourceIF = {
       }
     ]
   },
+  createRules: {
+    helpSection: {
+      header: `Help with Rules of the Association`,
+      helpText: {
+        section1: {
+          label: 'Each Cooperative Association must set its own rules to cover:',
+          items: [
+            'Governance',
+            'Overarching goals, needs, and actions to fulfill its purpose',
+            'Election of directors',
+            'Requirements for membership',
+            'Financial information management',
+            'Special rights and restrictions related to investment shares',
+            'How meetings are conducted'
+          ]
+        },
+        section2: {
+          label: 'The Rules need to:',
+          items: [
+            { type: 'text', value: `Balance the rights of individual members with the interests of the Cooperative
+              Association as a whole` },
+            { type: 'partialItems',
+              value: [
+                { type: 'text', value: 'Address the requirements of ' },
+                {
+                  type: 'link',
```

Comment:
I like it.

Please create an enum for 'link', 'text', etc.

Reply:
created enum(`ItemTypes`) and updated all relevant references to use this new enum